deps/softwarecontainer-dependencies.sh

```diff
@@ -44,16 +44,27 @@ function install {
 # For softwarecontainer
 install libdbus-1-dev libglibmm-2.4-dev libglibmm-2.4 \
         unzip bridge-utils lcov libjansson-dev libjansson4 \
-        dbus-x11 libcap-dev libtool python3-dev
+        dbus-x11 libcap-dev libtool python3-dev \
+        seccomp libseccomp-dev
```

Comment:
This is not mentioned in the commit message, so please put into separate commit to highlight that these features are enabled in the config.